src/grammar/brightscript.tmLanguage.spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import { Cache } from 'brighterscript/dist/Cache';
+import { createRegistry, parseGrammarTestCase, runGrammarTestCase } from 'vscode-tmgrammar-test/dist/src/unit/index.js';
+import { getErrorResultText } from './grammerTestHelpers.spec';
+import { standardizePath as s } from 'brighterscript';
+
+const brightscriptTmlanguagePath = s`${__dirname}/../../syntaxes/brightscript.tmLanguage.json`;
+
+describe('brightscript.tmlanguage.json', () => {
+    it('bool assignment', async () => {
+        await testGrammar(`
+             thing = true
+            '^^^^^ entity.name.variable.local.brs
+            '      ^ keyword.operator.brs
+            '        ^^^^ constant.language.boolean.true.brs
+        `);
+    });
+});
+
+const registries = new Cache();
+async function testGrammar(testCaseText: string) {
```

Comment:
This is the key part of this PR. The test wires up the `vscode-tmgrammar-test` module and then runs the test against the given string.